app/models/available_point.rb

```diff
@@ -85,7 +85,8 @@ def award_to(user, submission = nil)
       course_id: exercise.course_id,
       name: name,
       user_id: user.id,
-      submission: submission
+      submission: submission,
+      created_at: submission.created_at
```

Comment:
Actually, it would seem that ruby styleguide (https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas) doesn't like trailing commas, so no need to add them anywhere :stuck_out_tongue: